frontend/src/views/Register.vue

```diff
@@ -9,35 +9,46 @@
           <v-card-text>
               <v-form @submit.prevent="register()" v-model="valid">
 
-                  <v-text-field
-                    v-model="email"
-                    :rules="emailRules"
-                    label="E-mail"
+                  <v-text-field class="username"
+                    label="Username"
+                    prepend-icon="person"
                     required
+                    v-model="username"
+                    :rules="[(v => !!v || 'Username is required')]"
                   ></v-text-field>
 
-                  <v-text-field
-                    v-model="password"
+                  <v-text-field class="password"
                     label="Password"
-                    type="password"
+                    :type="passwordShow ? 'text' : 'password'"
+                    prepend-icon="lock"
+                    :append-icon="passwordShow ? 'visibility_off' : 'visibility'"
+                    @click:append="passwordShow = !passwordShow"
                     required
-                  ></v-text-field>
+                    v-model="password"
+                    :rules="[(v => !!v || 'Password is required')]"
+                 ></v-text-field>
 
-                  <v-text-field
-                    v-model="cpassword"
-                    label="Confirm Password"
-                    type="password"
-                    required
-                  ></v-text-field>
+                 <v-text-field class="password"
+                   label="Password"
+                   :type="cPasswordShow ? 'text' : 'password'"
+                   prepend-icon="lock"
+                   :append-icon="cPasswordShow ? 'visibility_off' : 'visibility'"
+                   @click:append="cPasswordShow = !cPasswordShow"
+                   required
+                   v-model="password"
```

Comment:
Make a new variable for the confirm password.

Reply:
cPassword is it's own variable

Reply:
never mind adding it as a private member